src/github_services.py

```diff
@@ -251,10 +270,9 @@ def create_discussion_comment(
     if discussion_id is None:
         raise Exception(f'{discussion_category} category is missing in GitHub Discussion.')
 
-
-    # Here, the below code written in GraphQL is being used to perform a mutation operation.
-    # More specifically, we are using it to comment in github discussion to let reviewers
-    # know about some of their pending tasks. To learn more, check this out
+    # The following code is written in GraphQL and is being used to perform a mutation
+    # operation. More specifically, we are using it to comment in GitHub discussion to
+    # let reviewers know about some of their pending tasks. To learn more, check this out
```

Comment:
check this out --> check out

(or, add a colon after "check this out")

Reply:
Added colon after "check this out".